pkg/oauthserver/oauthserver/auth.go

```diff
@@ -140,17 +143,11 @@ func (c *OAuthServerConfig) WithOAuth(handler http.Handler) (http.Handler, error
 	tokenRequestEndpoints := tokenrequest.NewEndpoints(c.ExtraOAuthConfig.Options.MasterPublicURL, c.getOsinOAuthClient)
 	tokenRequestEndpoints.Install(mux, urls.OpenShiftOAuthAPIPrefix)
 
-	// glog.Infof("oauth server configured as: %#v", server)
-	// glog.Infof("auth handler: %#v", authHandler)
-	// glog.Infof("auth request handler: %#v", authRequestHandler)
-	// glog.Infof("grant checker: %#v", grantChecker)
-	// glog.Infof("grant handler: %#v", grantHandler)
```

Comment:
Would these ever be useful as a glog.V(5) maybe?

Reply:
There is not enough useful information to justify it:

```
$ openshift start master |& grep auth.go
I0821 22:46:14.826574   15436 auth.go:146] oauth server configured as: &osinserver.Server{config:(*osin.ServerConfig)(0xc4210cc9a0), server:(*osin.Server)(0xc42192d860), authorize:osinserver.AuthorizeHandlers{(*handlers.AuthorizeAuthenticator)(0xc42153f0e0), (*handlers.GrantCheck)(0xc42153f140), (osinserver.AuthorizeHandlerFunc)(0x3d875d0)}, access:osinserver.AccessHandlers{(*handlers.AccessAuthenticator)(0xc42153f1a0)}, errorHandler:osinserver.defaultErrorHandler{}}
I0821 22:46:14.826600   15436 auth.go:147] auth handler: &handlers.unionAuthenticationHandler{challengers:map[string]handlers.AuthenticationChallenger{"basic-challenge":(*passwordchallenger.basicPasswordAuthHandler)(0xc420ba3d80)}, redirectors:(*handlers.AuthenticationRedirectors)(0xc420647360), errorHandler:(*errorpage.ErrorPage)(0xc420ba3d40), selectionHandler:(*selectprovider.SelectProvider)(0xc420647480)}
I0821 22:46:14.826623   15436 auth.go:148] auth request handler: &union.unionAuthRequestHandler{Handlers:[]authenticator.Request{(*session.Authenticator)(0xc420654380), (*basicauthrequest.basicAuthRequestHandler)(0xc42153ef00)}, FailOnError:false}
I0821 22:46:14.826641   15436 auth.go:149] grant checker: &registry.ClientAuthorizationGrantChecker{client:(*v1.oAuthClientAuthorizations)(0xc420ba2e50)}
I0821 22:46:14.826653   15436 auth.go:150] grant handler: &handlers.perClientGrant{auto:(*handlers.autoGrant)(0xa617d20), prompt:(*handlers.redirectGrant)(0xc420ba3de0), deny:handlers.emptyGrant{}, defaultMethod:"auto"}
```